packages/shared/src/components/modals/post/CreateSharedPostModal.tsx

```diff
@@ -151,32 +153,15 @@ export function CreateSharedPostModal({
           </Switch>
         )}
       </form>
+      <span className="flex flex-row items-center gap-2 px-4 tablet:hidden">
+        {footer}
+      </span>
```

Comment:
2 footers?

Reply:
This has `tablet:hidden` so it is not rendered on tablets.